public/locales/en.json

```diff
@@ -1,5 +1,5 @@
 {
-  "noWallet": "No Ethereum wallet found",
+  "noWallet": "No Binance Smart Chain wallet found",
```

Comment:
@Chef-Chungus FYI these file will be gone in future versions as we will start using Crowdin for the translations.